src/v2/api/group/GroupService.ts

```diff
@@ -131,4 +114,43 @@ export class GroupService {
 
     await this.studentRepository.update(user.id, {state: State.DECLINED});
   }
+
+  async getCaptain(groupId: string) {
+    const students = await this.groupRepository.getStudents(groupId);
+    for (const student of students) {
+      const roles = await this.studentRepository.getRoles(student.userId);
+      for (const role of roles){
+        if ( role.name == RoleName.CAPTAIN ){
+          return student;
```

Comment:
It's bad approach, because you send many internal data (i mean, when you send literally the database entity). you should cover it with new object, where you will input essential data. for example, userId, first, last, middle names, groupId, state.